index.html

```diff
@@ -0,0 +1,62 @@
+
+      <figcaption>
+        Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of
+        his
+        life-long war on hunger.
+      </figcaption>
```

Comment:
Please enclose the `figcaption` inside a `figure` element, The `figcaption` element is a caption describing the rest of the content of its parent `figure` element